pkg/runctl/config.go

```diff
@@ -76,25 +106,46 @@ func loadPipelineRunsConfig(clientFactory k8s.ClientFactory) (*pipelineRunsConfi
 		}
 		return nil, nil
 	}
-
+	var err error
 	if config.Timeout, err =
 		parseDuration(pipelineRunsConfigKeyTimeout); err != nil {
-		return nil, err
+		return err
 	}
 
 	if config.JenkinsfileRunnerPodSecurityContextRunAsUser, err =
 		parseInt64(pipelineRunsConfigKeyPSCRunAsUser); err != nil {
-		return nil, err
+		return err
 	}
 	if config.JenkinsfileRunnerPodSecurityContextRunAsGroup, err =
 		parseInt64(pipelineRunsConfigKeyPSCRunAsGroup); err != nil {
-		return nil, err
+		return err
 	}
 
 	if config.JenkinsfileRunnerPodSecurityContextFSGroup, err =
 		parseInt64(pipelineRunsConfigKeyPSCFSGroup); err != nil {
-		return nil, err
+		return err
 	}
 
-	return config, nil
+	return nil
+}
+
+func processNetworkMap(networkMap map[string]string, config *pipelineRunsConfigStruct) error {
+	defaultNetworkPolicyKey := networkMap[networkPoliciesConfigKeyDefault]
+	if defaultNetworkPolicyKey == "" {
+		return fmt.Errorf("no entry for default network policy key found")
+	}
+	if config.DefaultNetworkPolicy = networkMap[defaultNetworkPolicyKey]; config.DefaultNetworkPolicy == "" {
+		return fmt.Errorf("no network policy with key %q found", defaultNetworkPolicyKey)
+	}
+
+	networkPolicies := map[string]string{}
+	for key, value := range networkMap {
+		if key != defaultNetworkPolicyKey && !strings.HasPrefix(key, "_") {
+			networkPolicies[key] = value
+		}
```

Comment:
Also ignore the _empty string_ as key.

Reply:
See https://github.com/SAP/stewardci-core/pull/160/commits/e80520288c225c6abd78d0d40b7cade55515ec4c